pyre/extra_tests/parity_tests/flocals_writethrough_forcing_residual.py

```
@@ -0,0 +1,49 @@
+first recording is what this pins.  A second recording, reached much later,
+inlines ``setter`` instead of calling it; the store is then a residual that
+forces nothing at all, and that arm is a separate defect this does not cover.
+"""
+
+import sys
+
+ROUNDS = 1500
+
+
+def run(rounds):
+    def setter(value):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `setter`

Add return type annotation: `None`

(ANN202)